packages/common/src/eips.ts

```diff
@@ -471,4 +471,17 @@ export const EIPs: EIPsDict = {
     minimumHardfork: Hardfork.London,
     requiredEIPs: [],
   },
+  7516: {
+    comment: 'BLOBBASEFEE opcode',
+    url: 'https://eips.ethereum.org/EIPS/eip-7516',
+    status: Status.Draft,
+    minimumHardfork: Hardfork.London,
+    requiredEIPs: [4844],
+    gasPrices: {
+      blobbasefee: {
```

Comment:
You're right.  I loked across the `eips.ts` file and there's no consistency here.  Just a silly thought but maybe a follow-up PR is to just make all of the `gasPrices` fields in `common/src/eips` lowercase so we can drop that `toLowerCase` call here.  That's certainly a small performance gain for every single time we get the base fee.

Reply:
@acolytec3 Note that the line which @g11tech references to is in `codes.ts` of EVM (the opcode builder). The opcode builder expects all to be opcode names in common to be lowercase. Note: this method is only called if we get new opcodes (at EVM startup + fork transition) so this is not called each time for all opcodes if one is called.

Reply:
Won't die on this hill but in most cases we *do* have the camel case and I do somewhat like it for readability, but it is also true that this is a lot more inconsistent than I anticipated when looking through Common naming. Atm we can't update anyhow since this would be a breaking change.

So for here I guess we can keep both ways.